src/arguments.rs

```diff
@@ -116,6 +116,9 @@ pub struct Arguments {
     /// retrying partially failed downloads, or stitching the tiles with an external program.
     #[arg(short = 'c', long = "tile-cache")]
     pub tile_storage_folder: Option<PathBuf>,
+    /// A zoom_level image format suffix eg.default.png or default.jpg
+    #[arg(short = 'f', long = "zoom-format", default_value = "auto")]
+    pub zoom_format: String,
```

Comment:
If the option is specific to IIIF, then its name should probably reference it. 

Maybe something like `--iiif-tile-format=jpg`